src/main/kotlin/com/avail/compiler/splitter/Optional.kt

```diff
@@ -55,8 +58,7 @@ import java.util.*
 /**
  * An `Optional` is a [Sequence] wrapped in guillemets («»), and followed by a
  * question mark (?).  It may not contain [Argument]s or subgroups, and since it
- * is not a group it may not contain a [double
- * dagger][Metacharacter.DOUBLE_DAGGER] (‡).
+ * is not a group it may not contain a [double&#32;dagger][Metacharacter.DOUBLE_DAGGER] (‡).
```

Comment:
Long line.  You'll need to break it before "`[double`".

Reply:
Yep. Reflow does it correctly automatically, and produced the result that you described.